ZenPacks/zenoss/Import4/service_definition/imp4mariadb.json

```diff
@@ -143,14 +165,14 @@
        "Actions": null,
        "HealthChecks": {
          "running": {
-           "Script": "mysql -uroot -hlocalhost -P3306 -e 'select 1' > /dev/null",
+           "Script": "mysql --socket=/var/lib/mysql/mysql.sock -uroot -hlocalhost -P3306 -e 'select 1' && mysql --socket=/var/lib/mysql.model/mysql.sock -uroot -hlocalhost -P3307 -e 'select 1' > /dev/null",
            "Interval": 5.0
          }
        },
        "Prereqs": [
          {
-           "Name": "MariaDB Availability",
-           "Script": "su - zenoss -c '/opt/zenoss/bin/python /opt/zenoss/Products/ZenUtils/ZenDB.py --usedb zodb --execsql=\";\"'"
+           "Name": "MariaDB zodb and zep Availability",
+           "Script": "su - zenoss -c '/opt/zenoss/bin/python /opt/zenoss/Products/ZenUtils/ZenDB.py --dbport 3306 --usedb zep --execsql=\";\"' && su - zenoss -c '/opt/zenoss/bin/python /opt/zenoss/Products/ZenUtils/ZenDB.py --dbport 3307 --usedb zodb --execsql=\";\"' "
```

Comment:
Referring to the code on line 168, do you need to specify a host/port if you are using ```--socket```?